### PR TITLE
Add trailingslashit to the template locater

### PR DESCRIPTION
**Pull request by lukecarbis**
This PR makes it easier to change the template path directory, by switching to the `trailingslashit` function when locating templates, and adding documentation to the `block_lab_template_path` filter.

### CODE REVIEW

File: php/helpers.php [hunk 1]
```diff
@@ -165,7 +165,17 @@ function block_field_config( $name ) {
  * @return string|array
  */
 function block_lab_locate_template( $template_names, $path = '', $single = true ) {
-	$path            = apply_filters( 'block_lab_template_path', $path );
+	/**
+	 * Filters the path where block templates are saved.
+	 *
+	 * Note that template names are prefixed with the blocks directory.
+	 * e.g. `blocks/block-template.php`
+	 *
```
Comment (kienstra) [inline php/helpers.php:174]: Maybe this could also mention:

```
 * The logic below will look for the prefixed template name inside the $path.
```

**Comment by kienstra** [comment]
**Approved**

Hi @lukecarbis,
This looks good. Great idea to use `trailingslashit()`.

This method still got the templates as expected from the:

* Parent theme
* Child theme
* Template directory filtered with `'block_lab_template_path'`

**Review by kienstra — DISMISSED** [review]

File: php/helpers.php (hunk 1 above)
Comment (lukecarbis) [reply php/helpers.php:174]: Great idea. Added in https://github.com/getblocklab/block-lab/pull/358/commits/4cc09669bd1bb6343cbeed438b37c4fc96b1a52c.

**Comment by lukecarbis** [comment]
@kienstra I just added that extra comment line that you suggested. Could you please re-approve?

**Review by kienstra — APPROVED** [review]
**Approved**

Hi @lukecarbis,
Thanks for making that commit. This looks good.

**Pull request merged**